test/init/init.test.js

```diff
@@ -276,7 +278,7 @@ describe('init command', () => {
         });
 
         // Check if package.json is correctly configured
-        const pkgJson = require(join(assetsPath, 'package.json'));
+        const pkgJson = readFileContent(join(assetsPath, 'package.json'));
         expect(pkgJson.scripts['build'] === 'webpack --mode=production').toBeTruthy();
         expect(pkgJson.scripts['serve'] === 'webpack serve').toBeTruthy();
```

Comment:
Do we have command for build in `dev` mode?

Reply:
Nope

Reply:
Let's do it in separate PR, i think ideally will be great to have:
- `build`
- `build:dev`
- `build:prod` (same as `build`)
- `serve`
- `watch` (just watching without dev server)